src/Model/CourseManager.php

```diff
@@ -22,4 +22,18 @@ public function insert(array $item): void
         $statement->bindValue('capacity', $item['capacity'], \PDO::PARAM_INT);
         $statement->execute();
     }
+
+    public function update(array $item): void
+    {
+        $query = "UPDATE " . self::TABLE .
+            " SET name=:name, day=:day, time=:time, duration=:duration, capacity=:capacity WHERE id=:id";
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue('name', $item['name'], \PDO::PARAM_STR);
+        $statement->bindValue('day', $item['day'], \PDO::PARAM_STR);
```

Comment:
isn't the day supposed to be an int now in the database?

Reply:
yes it's true i changed this